Source/MovieContainer.swift

```diff
@@ -167,6 +181,7 @@ class MovieContainer: UIView {
     }
 
     func play() {
+
```

Comment:
I prefer to not have whitespaces here, Apple doesn't use them and I think we shouldn't too.

https://github.com/apple/example-package-deckofplayingcards/blob/master/Sources/Deck.swift